ja/controllers/request-response.rst

```diff
@@ -176,7 +176,7 @@ post 形式でデータを交換することがしばしばあります。 :php:
     $base = $request->getAttribute('base');
 
     // /subdir/ を保持
-    $base = $request->getAttribute('base');
+    $base = $request->getAttribute('webroot');
```

Comment:
$base -> $webroot

Could you please PR the correction of the English version first?